sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -75,6 +75,7 @@ dependencies {
   compile library.java.proto_google_common_protos
   compile library.java.protobuf_java
   compile library.java.slf4j_api
+  compileOnly library.java.hamcrest_library // should not appear in users' class path
```

Comment:
This module's `org.apache.beam.sdk.io.gcp.pubsub.TestPubsub` in "main" (not test) uses Hamcrest's matchers. Therefore declaring the dependency.

Reply:
provided would make more sense

Reply:
Updated to `provided`.